src/app/auth/components/signup/signup.component.ts

```diff
@@ -49,6 +50,9 @@ export class SignupComponent {
   agreedTerms = false;
   receiveUpdatesViaEmail = false;
 
+  passwordStrengthMessage: string = '';
+  passwordStrengthClass: string = '';
```

Comment:
In theory we can make this typing even more specific with a type union. That way we know that `Message` can only ever be "Strong", "Weak", etc.

In this case it might be a bit redundant but it's nice to keep that in mind, especially when testing for string equality like we do in this component.